rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -20,11 +20,22 @@ aws-smithy-http = { path = "../aws-smithy-http" }
 aws-smithy-types = { path = "../aws-smithy-types" }
 aws-smithy-json = { path = "../aws-smithy-json" }
 axum = { version = "0.3", features = [ "http1", "http2", "headers", "mime", "tower-log" ], default-features = false }
+async-trait = "0.1"
 bytes = "1.1"
+futures-util = { version = "0.3", default-features = false }
 http = "0.2"
 http-body = "0.4"
 hyper = { version = "0.14", features = ["server", "http1", "http2", "tcp"] }
 mime = "0.3"
+pin-project-lite = "0.2.7"
```

Comment:
From pin-project-lite documentation:
```This is the only reason to use this crate. However, if you already have proc-macro related dependencies in your crate's dependency graph, there is no benefit from using this crate. (Note: There is almost no difference in the amount of code generated between pin-project and pin-project-lite.)```

We already have proc-macro deps coming in with `async-trait` crate. Is there any other reason to use this instead of standard `pin-project` ?

Reply:
Nope. Switched to `pin-project`.